README.md

```diff
@@ -185,8 +177,6 @@ AED3 1134 9C23 A123 E5C4  AA4B 139C A045 3DA2 D773
 [64nightly]: https://github.com/qTox/qTox/releases/download/nightly/qtox-nightly-x86_64-release.exe
 [Flatpak]: https://github.com/qTox/qTox/releases/download/v1.17.6/qTox-v1.17.6.x86_64.flatpak
 [Flatpak nightly]: https://github.com/qTox/qTox/releases/download/nightly/qTox-nightly.flatpak
```

Comment:
The rest of these links should probably also be updated to point to toktok/qTox, or removed. :P

Reply:
Done, for the nightly ones. We'll update the release ones when we make a release.